spec/directives/mixin/content/arguments/scope.hrx

```diff
@@ -0,0 +1,74 @@
+<===> README.md
+Arguments passed to @content blocks are lexically scoped within those blocks.
+
+<===>
+================================================================================
+<===> no_argument/input.scss
+$var: top-level;
+
+@mixin mixin($var) {
+  mixin-var-before: $var;
+  @content;
+  mixin-var-after: $var;
+}
+
+a {
+  @include mixin(mixin-argument) {
+    content-var: $var;
+  }
+}
+
+<===> no_argument/output.css
+a {
+  mixin-var-before: mixin-argument;
+  content-var: top-level;
+  mixin-var-after: mixin-argument;
+}
```

Comment:
As above, since this isn't actually declaring or passing content arguments, it seems like it doesn't belong here.